### PR TITLE
MITRE Att&CK indicator: fixed word bound in indicator regex

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32268

## Description
fixed the word bound in the indicator regex.
tested manually, can't test in the build as it is not part of the core packs.

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Add tests

File: Packs/FeedMitreAttack/ReleaseNotes/1_1_9.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Indicator Types
+- **MITRE ATT&CK**
+Fixed an issue where the indicator matched invalid text.
```
Comment (amshamah419) [inline Packs/FeedMitreAttack/ReleaseNotes/1_1_9.md:4]:
```suggestion
Fixed an issue where an indicator would match on invalid text.
```

File: Packs/FeedMitreAttack/IndicatorTypes/reputation-mitreAttck.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "sortValues": null,
     "commitMessage": "",
     "shouldCommit": false,
-    "regex": "\\b((T\\d{3,4}\\.\\d{3})|(MOB-T|T|TA|G|S|M)\\d{3,4})|((attack-pattern–|intrusion-set|malware|tool|course-of-action|x-mitre-tactic|x-mitre-matrix)[a-z0-9]{8}(-[a-z0-9]{4}){3}-[a-z0-9]{12})|(((APT)|(CAPEC\\-))\\d{1,4})\\b",
+    "regex": "\\b((T\\d{3,4}\\.\\d{3})|(MOB-T|T|TA|G|S|M)\\d{3,4})\\b|\\b((attack-pattern–|intrusion-set|malware|tool|course-of-action|x-mitre-tactic|x-mitre-matrix)[a-z0-9]{8}(-[a-z0-9]{4}){3}-[a-z0-9]{12})|(((APT)\\b|\\b(CAPEC\\-))\\d{1,4})\\b",
```
Comment (yaakovi) [inline Packs/FeedMitreAttack/IndicatorTypes/reputation-mitreAttck.json:8]:
```suggestion
    "regex": "\\b((T\\d{3,4}\\.\\d{3})|(MOB-T|T|TA|G|S|M)\\d{3,4})\\b|\\b(((attack-pattern–)|intrusion-set|malware|tool|course-of-action|x-mitre-tactic|x-mitre-matrix)[a-z0-9]{8}(-[a-z0-9]{4}){3}-[a-z0-9]{12})\\b|\\b(((APT)|(CAPEC\\-))\\d{1,4})\\b",
```

**Review by reutshal — APPROVED** [review]
Nice

**Pull request merged**